docs/source/markdown/podman-create.1.md

```diff
@@ -22,6 +22,28 @@ Default settings for flags are defined in `containers.conf`. Most settings for
 remote connections use the server's containers.conf, except when documented in
 man pages.
 
+## IMAGE
+
+ The `image` uses a "transport":"details" format.
+
+ Multiple transports are supported when run locally.  Only the `docker` (container registry)
+ transport is allowed for remote access.
+
+  **dir:**_path_
+  An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+  **docker://**_docker-reference_
+  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$XDG_RUNTIME_DIR/containers/auth.json`, which is set using `(podman login)`. If the authorization state is not found there, `$HOME/.docker/config.json` is checked, which is set using `(docker login)`.
```

Comment:
"Accepts a reference to an image in a remote registry. The reference can include a path to a specific registry; if it does not, the registries listed in `registries.conf` will be queried to find a matching image. By default, credentials from `podman login` (stored at `$XDG_RUNTIME_DIR/containers/auth.json` by default) will be used to authenticate; if these cannot be found, we will fall back to using credentials in `$HOME/.docker/config.json`.